Lib/ast.py

```diff
@@ -626,7 +626,7 @@ def unparse(ast_obj):
     return unparser.visit(ast_obj)
 
 
-def main():
+def _main(args=None):
```

Comment:
please, revert the name change. technically someone might use it, it is a public name, and since `ast` does not have `__all__`. no need to break stuff for people without a good reason.